src/main/java/org/fundacion/jala/converter/models/facade/ParameterOutputChecksum.java

```diff
@@ -63,3 +63,4 @@ public String getFilename() {
         return filename;
```

Comment:
It should be fileName, file and name are separate words and according to lower camel case convention should be as mentioned before.

Reply:
It will be reviewed when ParameterOutoutChecksum is refactored.